source/events/cappCon2016/index.html.markdown

```diff
@@ -0,0 +1,169 @@
+---
+title: CappCon 2016
+author: Alexandre Wilhelm
+author_email: alexandre.wilhelmfr@gmail.com
+date: '2016-05-15'
+event_date: March 31st 2016
+speakers:
+  - name: Alexander Ljunberg
+    url: https://www.twitter/slevenbits
+  - name: Alexandre Wilhelm
+    url: https://www.twitter/WilhelmAlex
+  - name: Antoine Mercadal
+    url: https://www.twitter/primalmotion
+  - name: Christophe Serafin
+    url: https://www.twitter/@t00f67
+  - name: Martin Carlberg
+    url: https://www.twitter/mrcarlberg
+companies:
+  - name: Nuage Networks
+    url: http://www.nuagenetworks.net
+  - name: RWElephant
+    url: http://www.rwelephant.org
+  - name: Oops
+    url: http://www.oops.se
+  - name: University of Liege
+    url: https://www.ulg.ac.be
+---
+
+### About Event
+
+We're excited to announce the second ever full-blown conference dedicated to Cappuccino: CappCon 2016.
```

Comment:
We're very excited to announce the second conference entirely dedicated to Cappuccino: CappCon 2016!